src/Index.js

```diff
@@ -67,6 +68,7 @@ module.exports = {
   Styles: require('./constants/Style'),
 
   // App Utils
+  FocusManagement: require('./utils/FocusManagement'),
```

Comment:
Is there a specific reason why you're exposing `FocusManagement`? If not I'd say let's remove this line.

Reply:
I had thought I would need it internally but now that we have changed things I don't think I will.  I'll remove it.